python/tank/authentication/site_info.py

```diff
@@ -92,13 +92,24 @@ def reload(self, url, http_proxy=None):
         :param url:            Url of the site to query.
         :param http_proxy:     HTTP proxy to use, if any.
         """
+        # Check for valid URL
+        url_items = utils.urlparse.urlparse(url)
+        if (
+            not url_items.netloc
+            or url_items.netloc in "https"
```

Comment:
I'm really not sure about this line. How could that happen?

Reply:
Because the variable `url` is already sanitized by [sanitize_url](https://github.com/shotgunsoftware/tk-core/blob/master/python/tank/util/shotgun/connection.py#L273) it already have `https` prepended. So, when user is typing "https", URL will have `https://https`.